mythril/support/source_support.py

```diff
@@ -17,7 +15,7 @@ def __init__(
         self.source_type = source_type
         self.source_format = source_format
         self.source_list = source_list or []
-        self.meta = meta
+        self._source_hash = []
```

Comment:
What is the _source_hash variable used for. It seems like its overwritten later

Reply:
It can be used to query file index from hash.